Dockerfile

```diff
@@ -3,12 +3,11 @@ FROM python:3.13
 WORKDIR /code
 
 COPY ./requirements.txt .
+COPY ./src /code/src
```

Comment:
workdir is code, i dont think you need to specify it here, do the same as you did for requirements.txt